src/server.tsx

```diff
@@ -7,7 +7,7 @@ import { extractCritical } from 'emotion-server';
 import { renderHtmlDocument } from './utils/renderHtmlDocument';
 import { fetchDefaultEpicContent, fetchConfiguredEpicTests } from './api/contributionsApi';
 import { cacheAsync } from './lib/cache';
-import { ContributionsEpic } from './components/ContributionsEpic';
+import ContributionsEpic from './components/ContributionsEpic';
```

Comment:
Notice how we're importing the default here, not a named export. This is so we can access the 'component descriptor' that has the function that returns the JS.